From 2062786fe29110d501d986235d638014ffcef4e7 Mon Sep 17 00:00:00 2001
From: wep21 <daisuke.nishimatsu1021@gmail.com>
Date: Wed, 4 Jun 2025 18:02:05 +0900
Subject: [PATCH] include cstdint

Signed-off-by: wep21 <daisuke.nishimatsu1021@gmail.com>
---
 include/tins/ip_address.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/tins/ip_address.h b/include/tins/ip_address.h
index 5e6f6d1..c2f6c31 100644
--- a/include/tins/ip_address.h
+++ b/include/tins/ip_address.h
@@ -30,10 +30,10 @@
 #ifndef TINS_IPADDRESS_H
 #define TINS_IPADDRESS_H
 
+#include <cstdint>
 #include <string>
 #include <iosfwd>
 #include <functional>
-#include <stdint.h>
 #include <tins/cxxstd.h>
 #include <tins/macros.h>
 
-- 
2.39.5 (Apple Git-154)

